hbase-server/src/main/java/org/apache/hadoop/hbase/quotas/MasterQuotaManager.java

```diff
@@ -740,5 +740,14 @@ public void processFileArchivals(FileArchiveNotificationRequest request, Connect
       notifier.addArchivedFiles(filesWithSize);
     }
   }
+
+  /**
+   * Remove regionInfo entry for the table which is going to get dropped.
```

Comment:
Can you change this to: "Removes each region size entry where the RegionInfo references the provided TableName". A little more clear about what this method does (instead of the context in which you are calling it).

Reply:
done